### PR TITLE
COMMERCE-5375 fix flaky tests, make sure talend lastRunStartDate is null on first execution

### PR DESCRIPTION
**Pull request by igorbeslic**
Thank you Brian!!

### CODE REVIEW

**Comment by igorbeslic** [comment]
ci:test:sf

**Comment by igorbeslic** [comment]
ci:test:relevant

File: modules/apps/dispatch/dispatch-test/src/testIntegration/java/com/liferay/dispatch/internal/messaging/test/DispatchMessageListenerTest.java [hunk 1]
```diff
@@ -72,68 +76,107 @@ public void testDoReceiveOverlapAllowed() throws Exception {
...
 		List<DispatchLog> dispatchLogs =
 			_dispatchLogLocalService.getDispatchLogs(
 				dispatchTrigger.getDispatchTriggerId(), QueryUtil.ALL_POS,
 				QueryUtil.ALL_POS);
 
-		Assert.assertEquals(dispatchLogs.toString(), 4, dispatchLogs.size());
+		_assertExecutionSequence(dispatchLogs, executeCount, false);
+	}
+
+	private void _assertExecutionSequence(
```
Comment (igorbeslic) [inline modules/apps/dispatch/dispatch-test/src/testIntegration/java/com/liferay/dispatch/internal/messaging/test/DispatchMessageListenerTest.java:101]: @matijapetanjek here is new test method that checks execution sequence

**Review by dsitu — APPROVED** [review]

**Comment by igorbeslic** [comment]
ci:forward

**Pull request closed without merging**